src/tracks/playabletrack/wavetrack/ui/SpectrumView.h

```diff
@@ -34,11 +35,12 @@ class SpectralData{
     ,mStartT(std::numeric_limits<long long>::max())
     ,mEndT( 0 )
     ,mWindowSize( 2048 )
+    ,mHopSize ( 64 )
```

Comment:
Why not 512, which is 1/4 window size?

Why do you think you need more closely spaced hops?

Reply:
This is before our discussion on the rendering fix in Spectrogram, large hop size yields brush stroke that's not looking good.
Since we have found potential fix for that, will change it to 1/4 of the `mWindowSize`